apps/api-extractor/src/collector/ApiItemMetadata.ts

```diff
@@ -75,7 +75,19 @@ export class ApiItemMetadata {
    */
   public tsdocComment: tsdoc.DocComment | undefined;
 
-  // Assigned by DocCommentEnhancer
+  /**
+   * Tracks whether or not the associated API item is "undocumented".
```

Comment:
`Tracks whether or not the associated API item is known to be missing sufficient documentation.`

Reply:
```suggestion
   * Tracks whether or not the associated API item is known to be missing sufficient documentation.
```